packages/wallet-ui/src/services/useStarkNetSnap.ts

```diff
@@ -694,7 +722,11 @@ export const useStarkNetSnap = () => {
           },
         },
       });
-      return response;
+      if (balanceType === BalanceType.Spendable) {
+        return response.spendableBalance;
+      } else {
+        return response.totalBalance;
+      }
```

Comment:
IMO, the method should just return both response, else you may need to query twice to get both value, which is not make sense